.editorconfig

```diff
@@ -6,13 +6,7 @@ end_of_line = lf
 insert_final_newline = true
 indent_style = space
 charset = utf-8
-indent_size = 4
+indent_size = 2
```

Comment:
I did this because all JS is also indent of 2, so it appears that this should be the default so we can remove the override sections further down.